docs/excel/custom-functions-runtime.md

```diff
@@ -1,13 +1,15 @@
 ---
-ms.date: 09/20/2018
+ms.date: 09/27/2018
```

Comment:
Good to know that when we update we also update the date- I will keep this in mind